.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -18,7 +18,7 @@ body:
       required: true
   - type: input
     attributes:
-      label: What runtime/platform is your app running on?
+      label: What runtime/platform is your app running on? (with version if possible.)
```

Comment:
How about removing the `.` after `possible` because it's not necessary? This means make it to:

```
(with version if possible)
```

Reply:
thanks!